.github/workflows/prototype-tests-linux-gpu.yml

```diff
@@ -0,0 +1,96 @@
+name: Prototype tests on Linux
+
+on:
+  pull_request:
+
+jobs:
+  tests:
+    strategy:
+      matrix:
+        python-version:
+          - "3.7"
+          - "3.8"
+          - "3.9"
+          - "3.10"
+        gpu-arch-type: ["cpu"]
+        gpu-arch-version: [""]
+        runner: ["linux.2xlarge"]
+        include:
+          - python-version: "3.8"
+            gpu-arch-type: cuda
+            gpu-arch-version: "11.6"
+            runner: linux.4xlarge.nvidia.gpu
+      fail-fast: false
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    with:
+      job-name: Python ${{ matrix.python-version }}, ${{ matrix.gpu-arch-type }}
+      repository: pytorch/vision
+      gpu-arch-type: ${{ matrix.gpu-arch-type }}
+      gpu-arch-version: ${{ matrix.gpu-arch-version }}
+      runner: ${{ matrix.runner }}
+      timeout: 45
+      script: |
+        # Mark Build Directory Safe
+        git config --global --add safe.directory /__w/vision/vision
```

Comment:
@osalpekar we should probably just do this in the base `linux_job.yml` if everyone is having to recreate this

Reply:
Do we need it though? I've removed it in 119dc41b3ba01464f4bf2127fa5620084b74bae1 and the build is green: https://github.com/pytorch/vision/actions/runs/3731505086/jobs/6329814912. Under what circumstance do we need to? IIRC, this is only related to git submodules which we don't have. I guess it is not harmful, we can just do it by default.